modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/LogArea.scala

```diff
@@ -76,17 +94,30 @@ object LogArea {
    * Build the table log
    */
   def table(p: Props, s: State)(size: Size): VdomNode = {
+
+    // Custom renderers for the last column
+    val clipboardHeaderRenderer: HeaderRenderer[js.Object] = (_, _, _, _, _, _) =>
+      IconCopy.copyIcon(extraStyles = List(SeqexecStyles.logIconHeader))
+
+    val clipboardCellRenderer: CellRenderer[js.Object, js.Object, LogRow] = (_, _, _, row: LogRow, _) => {
+      // Simple csv export
+      // TODO use local tiems
+      val toCsv = s"${row.timestamp}, ${row.level}, ${row.msg}"
+      CopyLogToClipboard(toCsv)
+    }
+
     val columns = List(
       Column(Column.props(200, "timestamp", label = "Timestamp", disableSort = true)),
       Column(Column.props(80, "level", label = "Level", disableSort = true)),
-      Column(Column.props(size.width.toInt - 200 - 80, "msg", label = "Message", disableSort = true, flexGrow = 1))
+      Column(Column.props(size.width.toInt - 200 - 80 - 37, "msg", label = "Message", disableSort = true)),
```

Comment:
What are all those magic numbers?

Reply:
So the column's widths are defined in pixel and the message column get the rest to fill width. I suppose it would be better to have them as constants. I'll change that